staging/src/k8s.io/apiserver/pkg/server/width/list.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package width
+
+import (
+	"fmt"
+	"net/http"
+
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
+)
+
+// responsible for finding the "width" of LIST request(s)
+type listWidthFinder struct{}
```

Comment:
This seems to be unused now. So let's maybe remove completely and leave only the default one below.

Reply:
removed it